docs/template.readme

```diff
@@ -11,7 +11,7 @@
 
 ![insect](web/media/insect.png "insect - scientific calculator")
 
-A high precision scientific calculator with full support for physical units.
+A high precision scientific calculator with full support for physical units. Supports Node 10 and later.
```

Comment:
I don't mind, but why is that?

Reply:
That line is meant to describe the project in one single sentence.

I'm pretty sure that most Insect users have no idea what "Node" is, so I'd like to avoid confusing potential users here.